week-1-ts-ecosystem/w01e01-typescript-basics/src/utils/age.utils.ts

```diff
@@ -0,0 +1,5 @@
+import {User} from "../models"
+
+export function hasGivenAge(requiredAge: number): (user: User)=>boolean {
+    return (user: User): boolean => user.age >= requiredAge
+}
```

Comment:
Dla mnie te `utils` to `validators` i też nie pakowałbym ich (nie barrelował) w `index.ts` :)